### PR TITLE
update instruction for control plane

### PR DESCRIPTION
**Pull request by linsun**
data plane will be in a separate PR as it needs more testing

### CODE REVIEW

File: _docs/setup/kubernetes/upgrading-istio.md [hunk 1]
```diff
@@ -12,36 +12,47 @@ for the Kubernetes environment.
 
 ## Overview
 
-This guide describes how to upgrade an existing Istio deployment (including
-both control plane and sidecar proxy) to a new release of Istio. The upgrade
-process could involve new binaries as well as other changes like configuration
-and API schemas. The upgrade process may involve some service downtime.
+This guide describes how to upgrade an existing Istio deployment (including both control plane and sidecar proxy) to a new release of Istio. The upgrade process could involve new binaries as well as other changes like configuration and API schemas. The upgrade process may involve some service downtime.   To minimize the service downtime, please ensure your istio control plane components and your application are highly available with multiple replicas.
```
Comment (gyliu513) [inline _docs/setup/kubernetes/upgrading-istio.md]: `downtime.   To minimize ` 

Remove two spaces from above.

File: _docs/setup/kubernetes/upgrading-istio.md [hunk 2]
```diff
@@ -12,36 +12,47 @@ for the Kubernetes environment.
...
 Sidecar injector. We can use Kubernetes’ rolling update mechanism to upgrade the
-control plane components. It can be done by simply applying the new version
-yaml file directly, e.g.
+control plane components.  
+
+First, generate the desired istio control plane yaml file, e.g.
 
 ```command
-$ kubectl apply -f istio.yaml (or istio-auth.yaml)
+helm template  --set global.proxy.image=proxy \
```
Comment (gyliu513) [inline _docs/setup/kubernetes/upgrading-istio.md]: Should not here be `global.proxy.image=proxyv2`?

File: _docs/setup/kubernetes/upgrading-istio.md [hunk 3]
```diff
@@ -12,36 +12,47 @@ for the Kubernetes environment.
...
 to generate a customized Istio deployment, please use the customized yaml files
 generated by Helm instead of the standard installation yamls.
 
-The rolling update process will upgrade all deployments and configmaps to the
-new version. If there is any issue with the new control plane, you can rollback
-the changes either by applying the old version yaml files.
+Second, simply apply the new version of the desired istio control plane yaml file directly, e.g.
+
+```command
+$ kubectl apply -f istio.yaml (or istio-auth.yaml)
```
Comment (gyliu513) [inline _docs/setup/kubernetes/upgrading-istio.md]: I prefer follows, as people will copy the command.

```command
$ kubectl apply -f istio.yaml
```

```command
$ kubectl apply -f istio-auth.yaml
```

File: _docs/setup/kubernetes/upgrading-istio.md (hunk 1 above)
Comment (linsun) [reply _docs/setup/kubernetes/upgrading-istio.md]: ack

File: _docs/setup/kubernetes/upgrading-istio.md (hunk 2 above)
Comment (linsun) [reply _docs/setup/kubernetes/upgrading-istio.md]: no, because we want folks to use proxy for backwards compatible.

**Pull request merged**